src/main/java/tech/jhipster/lite/generator/server/springboot/dbmigration/flyway/infrastructure/primary/rest/FlywayResource.java

```diff
@@ -0,0 +1,32 @@
+package tech.jhipster.lite.generator.server.springboot.dbmigration.flyway.infrastructure.primary.rest;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import tech.jhipster.lite.generator.project.domain.Project;
+import tech.jhipster.lite.generator.project.infrastructure.primary.dto.ProjectDTO;
+import tech.jhipster.lite.generator.server.springboot.dbmigration.flyway.application.FlywayApplicationService;
+
+@RestController
+@RequestMapping("/api/servers/spring-boot/databases/migration/flyway")
+@Tag(name = "Spring Boot - Database")
+class FlywayResource {
+
+  private final FlywayApplicationService flywayApplicationService;
+
+  public FlywayResource(FlywayApplicationService flywayApplicationService) {
+    this.flywayApplicationService = flywayApplicationService;
+  }
+
+  @Operation(summary = "Add Flyway")
+  @ApiResponse(responseCode = "500", description = "An error occurred while adding Flyway")
+  @PostMapping
```

Comment:
It's because we'll have another API in this class to add user and authority sql files

```suggestion
  @PostMapping("/init")
```

Reply:
But the verb " POST " means that it's a creation ? So I think it's not useful, and we can use the verb " PUT " for update (add user and authority in this example) ?
I will do the change and use the "/init" to be "iso" with the existing.